src/OSPSuite.Core/Domain/Builder/EventGroupBuildingBlock.cs

```diff
@@ -6,7 +6,7 @@ public class EventGroupBuildingBlock : BuildingBlock<EventGroupBuilder>
    {
       public EventGroupBuildingBlock()
       {
-         Icon = IconNames.EVENT_GROUP;
+         Icon = IconNames.EVENT;
```

Comment:
I'm not sure if there's a good reason that this hasn't been changed. I also added a project conversion for this icon change.

Formerly, in MoBi, there was an object called ```BuildingBlockInfo``` that had the icon names. In almost all cases, it was just the icon name from the ```BuildingBlock``` with this one exception where "EventGroup" was not used and instead, "Event" was used.

That meant, that we have these icons ```EventRed``` and ```EventGreen```.

I changed this just to be consistent - we always use the ```BuildingBlock``` icon name for our display icon.